### DISCUSSION TITLE
Error inserting data into Supabase

### DISCUSSION THREAD

**Discussion by rbrandon087** [supabase/supabase#26131] [Questions] [answered]
Question would I need a Auth Token in my form its just a career form being submitted no login I have added a RLS policy but I keep getting a error I checked my names to match my Db and submitting but im not sure. Thank you
![Capture](https://github.com/supabase/supabase/assets/86093645/02c81aff-6214-4b78-bd47-8c87c60ce5d0)
![policy](https://github.com/supabase/supabase/assets/86093645/d0bfd5cd-d75c-4631-a7f8-35f5f336971c)

**Comment by GaryAustin1** [comment, marked as answer]
You asked this on discord also.  TO authenticated means you only allow signed in users.  You probably want TO public.  If you do .select() on the insert then you also have to meet select RLS.

Not sure that error has anything to do with this or Supabase.

Note:  You probably don't want .select() on the insert and you probably want the select policy to be at least TO authenticated so that everyone can't read the table entries.
